apps/meteor/client/views/admin/moderation/ModConsoleReportDetails.tsx

```diff
@@ -0,0 +1,45 @@
+import type { IUser } from '@rocket.chat/core-typings';
+import { Tabs, TabsItem, ContextualbarHeader, ContextualbarTitle } from '@rocket.chat/fuselage';
+import { type TranslationKey, useTranslation, useRouter } from '@rocket.chat/ui-contexts';
+import React, { useState } from 'react';
+
+import { Contextualbar, ContextualbarClose } from '../../../components/Contextualbar';
+import UserMessages from './UserMessages';
+import UserReportInfo from './UserReports/UserReportInfo';
+
+type ModConsoleReportDetailsProps = {
+	userId: IUser['_id'];
+	default: string;
+	onRedirect: (mid: string) => void;
+};
+
+const tabs = ['Messages', 'Users'];
+
+const ModConsoleReportDetails = ({ userId, default: defaultTab, onRedirect }: ModConsoleReportDetailsProps): JSX.Element => {
```

Comment:
```suggestion
const ModConsoleReportDetails = ({ userId, default: defaultTab, onRedirect }: ModConsoleReportDetailsProps) => {
```

Reply:
Thank you, done!
Could you please help me understand why this is not something expected? I would appreciate your help

Reply:
It's not necessary to tell you're going to return a ReactElement, it's identified automatically

Reply:
I see, interesting, thank you!